packages/react-native-renderer/src/ReactNativeFrameScheduling.js

```diff
@@ -27,8 +26,6 @@ const frameDeadlineObject: Deadline = {
 };
 
 function setTimeoutCallback() {
-  isCallbackScheduled = false;
```

Comment:
Why'd you remove `isCallbackScheduled`?

Doesn't this mean that >1 call to `scheduleDeferredCallback` will now result in a timer that can't be cancelled?

Wouldn't it be safer to just store a `callbackID` and check for it to be non-null before scheduling? (Then if it is non-null, we can just return it and skip the additional timeout)?

Reply:
> Doesn't this mean that >1 call to scheduleDeferredCallback will now result in a timer that can't be cancelled?

We don't ever call it more than once. If we want to be defensive I'd prefer to throw. ReactNoop does this, so I'm pretty confident we won't regress.

Reply:
> We don't ever call it more than once.

I don't like the fact that the implementation of this part of the code has been changed to depend on how another piece of the code uses it. That feels brittle.

If you'd rather throw, that's cool. I'd support that.